torch/optim/radam.py

```diff
@@ -44,6 +44,10 @@ class RAdam(Optimizer):
 
     For further details regarding the algorithm we refer to `On the variance of the adaptive learning rate and beyond`_.
 
+    This implementation uses the same weight_decay implementation as Adam (were the weight_decay is applied
+    to the gradient) and not the one from AdamW (were weight_decay is applied to the update). Note that, for
+    historical reasons, this is different from the `original author implementation`_.
```

Comment:
I'm not sure tbh. It does match the other optimizers and is not specifically specified in the paper. So I guess that this was the default implementation we came up with.
But I don't know for sure so I left this vague. You think I should investigate the history to figure this out?

Reply:
I imagine you have more valuable things to do, but "historical reasons" is so vague I wonder if it's worth mentioning at all? This could just say, "This is different from..."